README.md

```diff
@@ -7,3 +7,13 @@ This repo is the equivalent of the [Anaconda msgpack-cxx](https://anaconda.org/c
 
 **NOTE:** Packages created by this repo only contain the raw source code and not the Python package.
 The Python package can be found in [msgpack](https://pypi.org/project/msgpack/).
+
+Credits go to the maintainers of the [msgpack-cxx](https://github.com/msgpack/msgpack-c/tree/cpp_master) package.
+
+## License
+
+The files in this repository are licensed under the Boost Software License - Version 1.0.
+
+The distributed msgpack-cxx package is a redistribution of [msgpack-cxx](https://github.com/msgpack/msgpack-c/tree/cpp_master) and therefore falls under the [msgpack-cxx license model](https://github.com/msgpack/msgpack-c/blob/cpp_master/COPYING) as described in the [COPYING file in the msgpack-cxx](https://github.com/msgpack/msgpack-c/blob/cpp_master/COPYING).
+Since all distributed files are obtained by download in the automation, instead of being explicitly included into this library,
```

Comment:
this is not true anymore due to `src/msgpack_cxx/__init__.py`. can you please revise this block?